pywebostv/model.py

```diff
@@ -20,3 +20,14 @@ def __getitem__(self, val):
 
     def __repr__(self):
         return "<InputSource '{}'>".format(self["label"])
+
+
+class AudioOutputSource(object):
+    def __init__(self, data):
+        self.data = data
+
+    def __getitem__(self):
+        return self.data
+
```

Comment:
If we don't have any interesting fields in the response (or in other words, we only care about the main string like "tv_output" etc), we can get rid of this entire `__getitem__`. This class doesn't have any dict-like access requirements.

Reply:
Okay, thanks again for your kind input! I've just recently started getting into python and looking forward to anything I can learn!

Reply:
Ohh, you are super awesome for being able to navigate through the sources on your own esp when you're a beginner :)

Reply:
Thanks, really appreciate it!!